starter-code/css/main.css

```diff
@@ -3,6 +3,149 @@ Colors:
 
 Text: 1A1A1A
 Green: #00B172
-White: #FFF
+White: #FFF*/
+
+nav{
+    position: fixed;
+    display: flex;
+    justify-content: space-between;    
+    width: 96%;
+    padding: 10px 2%;
+    background-color: white;
+    align-items: center;
+    font-size: 25px;
+  }
+   nav img{
+    width: 19%;
+    height:243px;
+    float: left;
+   }
+ 
+#menu{
+    display: flex;
+  }
+
+
+#menu  li{
+    display: flex;
+    justify-content: space-around;
+    font-size: 70px;
+    list-style: none;
+    margin: 60px;
+    padding-top: 16px
+ 
+}
+header{
+    height: 2100px;
+}
+#container{
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 850px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: white;
+}
+#centered{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform:translate(-50%,-50%);
+}
+h1{
+    font-size: 238px;
+    text-align: center;
+}
+h3{
+    
+    font-size: 75px;
+    text-align: center;
+}
+h4{
+    
+    font-size: 75px;
+    text-align: center;
+}
+#sectionone{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-bottom: 5%;
+  }
+  
+  #content{
+
+        padding-top: 50px;
+        width: 105%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-size: 55px;
+      }
+  #threeParts{
+      display: flex;
+      justify-content: space-evenly;
+      align-items: baseline;
+      align-content: center;
+      text-align: center;
+      margin-left: 20px;
+  }
+  #threeParts img{
+      width: 65%;
+  }
+#threeParts h3{
+    color: #00B172;
+    font-size: 82px;
+
+}
+#threeParts p{
+    font-size: 60px;
+}
+.sectionTwo {
+    background-color: #00B172;
+    background-image: url("file:///home/adele/code/labs/lab-css-spotify-clone-master/starter-code/images/spotify-icon-white.png");
```

Comment:
when using images, you should provide a relative link based on the file you are in

```
background-image: url("./images/spotify-icon-white.png")
```